global.json

```diff
@@ -0,0 +1,5 @@
+{
+  "sdk": {
+    "version": "8.0.100-rtm.23519.30"
```

Comment:
I updated the SDK to pull in https://github.com/dotnet/aspnetcore/pull/51134/commits/425bd12a5a442711271761df91b580c785f38fa5 which fixed antiforgery tokens rendered with server interactivity. This only affects the logout form.